frontend/src/lib/neuroagent_types.ts

```diff
@@ -0,0 +1,3398 @@
+/**
```

Comment:
IMO we could add a new job in the CI that checks that this file is up to date. The same logic as we have for the `/tools/autogenenerated_types/*`.

I am just afraid that unless we somehow enforce this file will not be up to date.


Also, any chance you can put the command to run in some `README.md`?